src/components/ui/Dropdown/Dropdown.tsx

```diff
@@ -5,7 +5,7 @@ export type DropdownPosition = "top" | "bottom";
 
 export function Dropdown(props: React.HTMLAttributes<HTMLDivElement>) {
   const ref = React.useRef<HTMLDivElement>(null);
-  const [position, setPosition] = React.useState<DropdownPosition>("bottom");
+  const [position] = React.useState<DropdownPosition>("bottom");
```

Comment:
Not sure why this has been kept as a state variable if this value isn't being changed. If this is a static value, you can use a simple variable right?

Reply:
yes